### PR TITLE
[Alert] Add validation message across the alert form

### PR DESCRIPTION
**Pull request by AdityaHegde**
Improve the validation in alerts create/edit.
- [x] Run validate on fields per tab when `Next` is pressed.
- [x] Show validation for a criteria at the bottom.

### CODE REVIEW

File: web-common/src/components/forms/InputV2.svelte [hunk 1]
```diff
@@ -57,21 +58,21 @@
...
     on:keydown={handleKeyDown}
-    on:focus={() => (focus = true)}
-    on:blur={() => (focus = false)}
-    bind:this={inputElement}
-    bind:value
+    {placeholder}
+    type="text"
   />
-  {#if error && !focus && value}
+  {#if error && (alwaysShowError || (!focus && value))}
```
Comment (ericpgreen2) [inline web-common/src/components/forms/InputV2.svelte:75]: This is getting complicated. Rather, I think we should:
- Use just `{#if error}`
- At the form-level, validate the field on every keystroke

File: web-common/src/features/alerts/criteria-tab/parseCriteriaError.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// criteria[0].value must be a `number` type, but the…inal value was: `NaN` (cast from the value `""`).
```
Comment (ericpgreen2) [inline web-common/src/features/alerts/criteria-tab/parseCriteriaError.ts:1]: Typo: "the...inal"

**Comment by ericpgreen2** [comment]
A couple UX findings:

1. Here I was able to press "Next" to advance to the next screen, even though the criteria didn't pass validation.<br>
image


2. Are alert names allowed to be numbers? I wasn't getting any validation error here:<br>
image

File: web-common/src/features/alerts/criteria-tab/parseCriteriaError.ts (hunk 2 above)
Comment (AdityaHegde) [reply web-common/src/features/alerts/criteria-tab/parseCriteriaError.ts:1]: This was just for my reference. Will remove this comment.

File: web-common/src/components/forms/InputV2.svelte (hunk 1 above)
Comment (AdityaHegde) [reply web-common/src/components/forms/InputV2.svelte:75]: Hmm i was not a 100% sure what the other checks were. So i dont want to unintentionally break other features.

**Review by ericpgreen2 — APPROVED** [review]

**Pull request merged**